example/settings.gradle.kts

```diff
@@ -60,3 +60,24 @@ fun String.includeProject(id: String, path: String = "$this/$id") {
   includeProject("sdf3.eclipse")
   includeProject("sdf3.intellij")
 }
+
+"multilang/spoofaxcore".run {
```

Comment:
Maybe I'm missing something, but these projects do not seem to be included in the PR?